libflux/flux-core/src/semantic/types.rs

```diff
@@ -217,6 +228,7 @@ impl fmt::Display for Error {
 
 /// `Kind` represents a class or family of types.
 #[derive(Debug, Display, Clone, Copy, PartialEq, Eq, Hash)]
+#[allow(missing_docs)]
```

Comment:
Kind represents a constraint on a type variable to a specific kind (i.e. type class).